src/CameraControls/useCenterGraph.ts

```diff
@@ -178,12 +178,6 @@ export const useCenterGraph = ({
           // Center the graph once nodes are loaded on mount
           await centerNodes(nodes, { animated: false });
           mounted.current = true;
-        } else {
```

Comment:
Wait - so now it won't auto center?

Reply:
It still auto centers on selection and mount, but not whenever nodes change. I was thinking about something like a deep compare to only trigger the centering if one of the `node.position`'s changed, but then dragging while zoomed is still an issue.

The dragging issue makes this feel hard enough to predict when or when not to re-center that I figured leaving it to be handled outside the canvas was the safest move